pkg/kube/inject/testdata/inject/gateway.yaml.injected

```diff
@@ -111,7 +111,13 @@ spec:
             port: 15021
           periodSeconds: 15
           timeoutSeconds: 3
-        resources: {}
+        resources:
+          limits:
+            cpu: "2"
```

Comment:
should we removed default cpu limit?
- https://medium.com/omio-engineering/cpu-limits-and-aggressive-throttling-in-kubernetes-c5b20bd8a718
- https://www.howtogeek.com/devops/should-you-set-kubernetes-cpu-limits/
- https://home.robusta.dev/blog/stop-using-cpu-limits